src/pages/lessons/[slug].tsx

```diff
@@ -203,7 +203,10 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
         }}
       />
       <Head>
-        <script src="//cdn.bitmovin.com/player/web/8/bitmovinplayer.js" />
+        <script
```

Comment:
since the player component isn't onscreen it won't load the script, so I thought that this would load the bitmovin js earlier